common/network-common/src/test/java/org/apache/spark/network/util/TransportFrameDecoderSuite.java

```diff
@@ -153,7 +148,7 @@ public void testEmptyFrame() throws Exception {
   @Test(expected = IllegalArgumentException.class)
   public void testLargeFrame() throws Exception {
     // Frame length includes the frame size field, so need to add a few more bytes.
-    testInvalidFrame(Integer.MAX_VALUE + 9);
+    testInvalidFrame((long) Integer.MAX_VALUE + 9);
```

Comment:
To unblock you, can you add `@Ignore` and file a bug to fix or remove this test?